src/pages/home/report/ReportActionContextMenu/ReportActionContextMenu.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {Pressable, View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../../../styles/styles';
+import getReportActionContextMenuStyles from '../../../../styles/getReportActionContextMenuStyles';
+import getButtonState from '../../../../libs/getButtonState';
+import ContextActions from './ContextActions';
+import variables from '../../../../styles/variables';
+import Tooltip from '../../../../components/Tooltip';
+
+const propTypes = {
+    // eslint-disable-next-line react/no-unused-prop-types
+    reportID: PropTypes.number.isRequired,
+    // eslint-disable-next-line react/no-unused-prop-types
+    reportActionID: PropTypes.number.isRequired,
+    isMini: PropTypes.bool,
+    shouldShow: PropTypes.bool,
+};
+
+const defaultProps = {
+    isMini: false,
+    shouldShow: false,
+};
+
+const ReportActionContextMenu = (props) => {
+    const {wrapperStyle, getButtonStyle} = getReportActionContextMenuStyles(props.isMini);
+    return props.shouldShow && (
+        <View style={[
+            ...wrapperStyle,
+            styles.flex1,
+        ]}
+        >
+            {ContextActions.map(((contextAction) => {
+                const Icon = contextAction.icon;
+                return (
+                    <Tooltip
+                        title={contextAction.text}
+                        key={contextAction.text}
+                        aria-label={contextAction.text}
```

Comment:
Can native use `aria-label`?

Reply:
I don't think so ... but if you look at `src/components/Tooltip/index.native.js`, none of these props are used 😉

Reply:
It just takes the props and renders the children

Reply:
Hmm ok still seems like platform specific code

Reply:
Yeah, AFAIK nothing bad happens if I remove this, so I can if you feel strongly. But `aria-label` in general just helps make the app more accessible to screen-readers, right? And even though we haven't done a deep dive on accessibility yet, that seems like a good thing.

My take is that, yes, it is platform specific; but so is the tooltip itself. We use file extensions in the tooltip module to mask the platform inconsistencies - just like all our other platform specific code. I follow a similar pattern for `Hoverable`. It's used in `ReportActionItem`, but you just have to trust/be aware that it will have no effect on native platforms, which should be apparent if you look at the `Hoverable` module.

Reply:
No worries. Gonna try to explain my concern better here and let me know if you still disagree.

- Using `aria-label` in cross-platform code might encourage the use of `aria-label` in other cross-platform code. 
- Even though we are not rendering any `Tooltip` on native platforms we should still avoid passing **html attributes** in cross-platform code. This will encourage that philosophy that the app is "cross-platform" not introduce web specific ideas.
- e.g. if we added an `aria-label` to a `View` next it might throw an error or warning on native but if we wanted the ability to do that we might instead create a `View` that accepted a `label` prop and then have the web side use the label for an `aria-label` and mobile do whatever it would do.

Reply:
Okay, good points. I'll nix this and leave this for a future concern in the accessibility project.